### DISCUSSION TITLE
Speedtest tracker LXC

### DISCUSSION THREAD

**Discussion by Koliera** [tteck/Proxmox#1678] [Ideas]
Was wondering if there's a way to set up a small LXC on the following - https://github.com/alexjustesen/speedtest-tracker
It would be interesting to have stats on the network speed over some time and this repo looks to be one of the few ones not abandoned for this purpose.

**Comment by Koliera** [comment]
Bump

**Comment by axsrza** [comment]
About the project above, we have **alexjustesen** continued with the project, I would like to see a script like this for LXC.

https://github.com/alexjustesen/speedtest-tracker 

UP!

**Comment by andrebnc** [comment]
I would like to support this proposal.

**Comment by franz19811** [comment]
Gostaria de apoiar a proposta tambem.

**Comment by det-peralta** [comment]
I would like to support this proposal.

**Comment by titof2375** [comment]
I would like to support this proposal.

**Comment by Fabick75** [comment]
Hi !
About this project, I wrote a simple guide in the discussion section of the project, to use an LXC container with this piece of software. Because the project is dockerized, I've done all the steps to install on a debian container under proxmox, without using docker.

This is the link to the discussion : https://github.com/alexjustesen/speedtest-tracker/discussions/1131

**Comment by axsrza** [comment]
obrigado, @tteck https://tteck.github.io/Proxmox/#myspeed-lxc

**Reply by shuperluu** [reply]
I installed this too and is awesome. Thanks to it discovered the best time of the week to perform some cloud-local syncs faster, based on some trends easy to catch by eye.

**Comment by trackmastersteve** [comment]
I'd love this!
